themes/godotengine/pages/governance.htm

```diff
@@ -0,0 +1,84 @@
+title = "Governance"
+url = "/governance"
+layout = "more"
+description = "Governance model"
+is_hidden = 0
+==
+{##}
+{% put title %} Governance {% endput %}
+
+<div class="container">
+
+  <h3>Legal status</h3>
+  <p>
+    Godot is legally a part of the
+    <a href="https://sfconservancy.org">Software Freedom Conservancy</a>. As a
+    not-for profit charity, the Software Freedom Conservancy receives donations
+    and, based on their mission statement, use them for the benefit of the
+    project.
+    <strong>As no company is behind Godot, it can't be sold or purchased.</strong>
+  </p>
+  <p>
+    Within the organization sits the Godot
+    <a href="/contact#plc">Project Leadership Comittee</a>. This committee
+    decides and authorizes the usage of project funds (donations and grants).
+  </p>
+  <p>
+    <a href="/license">Godot's license is MIT</a>. This means you can do pretty
+    much everything with it, as long as you don't misrepresent the origin of the
+    software. In practice, even with that condition in place, you can do
+    anything you want as if you wrote Godot yourself.
+  </p>
+  <p>
+    Still, the MIT license allows creating proprietary derivatives.
+    Many companies have private forks of the engine to suit their own needs, but
+    what is open will remain open forever. <strong>No legal way exists to cease
+    your right to use Godot.</strong> It's as if it was your own property;
+    nobody can take it from you.
+  </p>
+  <p>
+    Godot is made by thousands of skilled contributors who donate their free
+    time (some of them are paid from donations). Even if some contributors
+    suddenly decided to push new Godot versions as proprietary, it would be
+    difficult for them to compete or be as productive as the rest. Godot is a
+    true community effort, created and owned by everyone.
+  </p>
+
+  <h3>Project leadership, fund usage and decisions</h3>
+  <p>
+    While Godot has a <a href="/contact#plc">Project Leadership Committee</a>,
+    <strong>there is no central authority deciding what gets done</strong>.
+    The Project Leadership Committee rarely decides something on its own.
+    Usage of funds is discussed and approved by the
+    <em><a href="https://github.com/orgs/godotengine/people">advisors</a></em>,
```

Comment:
Not everyone on that list is formally part of the "advisors", and some advisors are not included in the list as they likely didn't make their @godotengine member status public. (Check the page in private mode.)

So until we have a clear listing, it's probably best not to include this link.

Reply:
This is still not resolved.